docs/jwst/cube_build/main.rst

```diff
@@ -57,6 +57,20 @@ a single NIRSpec SCA. In high resolution mode the 30 slices are imaged on the tw
 Terminology
 -----------
 
+General IFU Terminology
++++++++++++++++++++
```

Comment:
The line of pluses here needs to be as long as the text above, otherwise sphinx will complain (and give bad results).

Reply:
derp, fixed.